Nodejs/Product/Nodejs/Project/ImportWizard/FileSourcePage.xaml

```diff
@@ -52,12 +52,5 @@
                 </ListView.ItemTemplate>
             </ListView>
         </wpf:LabelledControl>
-        
-        <wpf:LabelledControl Title="Select any other options for your project."
-                           Grid.Row="1">
-            <StackPanel Orientation="Vertical">
-                <CheckBox IsChecked="{Binding ExcludeNodeModules}">Exclude node__modules</CheckBox>
```

Comment:
You should remove the ExcludeNodeModules binding as well if it's no longer being used.